modules/metastorage-client/src/integrationTest/java/org/apache/ignite/internal/metastorage/client/ItMetaStorageServiceTest.java

```diff
@@ -820,7 +823,80 @@ public void testMultiInvoke() throws Exception {
                 new StatementResult(false))));
 
         assertThat(resultIf.orElse().update(), upd(new Update(
-                Arrays.asList(new org.apache.ignite.internal.metastorage.server.Operation(OperationType.PUT, key2.bytes(), rval2)),
+                List.of(new org.apache.ignite.internal.metastorage.server.Operation(OperationType.PUT, key2.bytes(), rval2)),
+                new StatementResult(false))));
+    }
+
+    @Test
+    public void testMultiInvokeValueConditions() throws Exception {
```

Comment:
Due to the integration nature of this suite - I think we should avoid creation of many test methods with different combination of Conditions here. Conditions should be tested in according unit tests suites.

But, I see, that you want to check integration of all parts for new value conditions - I think the good idea, just to extend current `testMultiInvoke` with the needed cases.

Reply:
done